README.md

```diff
@@ -100,7 +100,7 @@ If you’d like to add a cryptocurrency to BasicSwap, either [apply for a listin
 
 ### Chats
 
-* **For developers** The chat [#particl-dev:matrix.org](https://app.element.io/#/room/#particl-dev:matrix.org) using [Element](https://element.io) (formerly Riot).
+* **For developers** The chat [#particl-dev:matrix.org](https://matrix.to/#/#particl-dev:matrix.org) using a Matrix client.
 * **For community** The community chat [https://discord.me/particl](https://discord.me/particl) [![Discord](https://img.shields.io/discord/391967609660112925)](https://discord.me/particl).
```

Comment:
add these i think (to community)

general
https://matrix.to/#/%23particl:matrix.org
basicswap
https://matrix.to/#/%23basicswap:matrix.org